products/statement-generator/src/components/PrimaryLandingSection.tsx

```diff
@@ -0,0 +1,117 @@
+    const [matches, setMatches] = useState(false);
+
+    useEffect(() => {
+        const media = window.matchMedia(query);
+        if(media.matches !== matches) {
+            setMatches(media.matches);
+        }
+        const listener = () => setMatches(media.matches);
+        window.addEventListener("resize", listener);
+        return () => window.removeEventListener("resize", listener);
+    }, [matches, query]);
+
+    return matches;
+};
+
+const useStyles = makeStyles<Theme, StyleProps>(({ palette, typography, spacing, breakpoints }) =>
+    createStyles({
+        root: {
+            background: palette.secondary.main,
+            color: palette.primary.darker,
+            minHeight: '686px',
+            display: 'flex',
+            flexDirection: 'row',
+            padding: spacing(3),
+        },
+        content: {
+            // height: '100%',
+            paddingLeft: spacing(3),
+            paddingRight: spacing(3),
+            paddingTop: spacing(2),
+            paddingBottom: spacing(2),
+            maxWidth: '1300px',
+            display: 'flex',
+            flex: 1,
+            flexFlow: 'row',
+            margin: '0 auto',
+            alignItems: 'center',
+        },
+        title: {
+            fontFamily: typography.fontFamily,
+            fontSize: '40px',
+            
+            [breakpoints.down('sm')]: {
+                fontSize: '24px',
+                lineHeight: '117%'
+            }
+        },
+        span: {
+            color: palette.primary.main,
+        },
+        text: {
+            alignItems: 'flex-end'
+        },
+        subtitle: {
+            fontFamily: typography.fontFamily,
+            fontSize: '25px',
+            lineHeight: '160%',
+            
+            [breakpoints.down('sm')]: {
+                fontSize: '20px',
+                width: '100%'
+            }
+        },
+        button: {
+            alignSelf: 'flex-start',
+            paddingTop: spacing(3),
+            [breakpoints.down('sm')]: {
+                margin: '0 auto'
+            }
+        }
+    })
+)
+
+const PrimaryLandingSection = ({ goNextPage }: ComponentProps) => {
+    const classes = useStyles({ theme: 'light' });
+    const landingImage = "https://via.placeholder.com/590x350";
+    const landingImageAlt = 'A placeholder waiting for the correct image';
+    const isMobile = useMediaQuery('(max-width:1236px)');
+
+    return (
+        <section className ={classes.root}>
+            <div className =  {classes.content}>
+                <ContentContainer className={classes.text}>
+                    <h1 className={classes.title}><span className={classes.span}>Expunge Assist</span> accelerates the <span className={classes.span}>Record Clearance</span> process by helping user generate a declaration letter</h1>
+                    {isMobile ? (<ContentContainer><img src={landingImage} alt={landingImageAlt}/></ContentContainer>) : <></>}
```

Comment:
This shouldn't need to be wrapped around `<ContentContainer>` but it looks like it we'll have to adjust the image size in mobile view :(
![Screen Shot 2022-02-01 at 3 50 16 PM](https://user-images.githubusercontent.com/2266545/152070654-2b731751-2786-4233-af61-17c42a0fb615.png)